nilearn/decoding/tests/test_space_net.py

```diff
@@ -187,6 +188,20 @@ def test_tv_regression_3D_image_doesnt_crash():
                      penalty="tv-l1", is_classif=False, max_iter=10).fit(X, y)
 
 
+def test_graph_net_classifier_score():
+    iris = load_iris()
+    X, y = iris.data, iris.target
+    y = 2 * (y > 0) - 1
+    X_, mask = to_niimgs(X, (2, 2, 2))
+    gnc = SpaceNetClassifier(mask=mask, alphas=1. / .01 / X.shape[0],
+                             l1_ratios=1., tol=1e-10,
+                             penalty="graph-net", loss='logistic',
+                             standardize=False, verbose=0,
+                             screening_percentile=100.).fit(X_, y)
+    accuracy = gnc.score(X_, y)
+    assert_less_equal(0, accuracy)
```

Comment:
Could the test be a bit more specific ?
Could you add a test to ensure that  the new loss is actually used by default ?